docs/dashboard/README.md

```diff
@@ -0,0 +1,32 @@
+<!--
+title: Serverless - Dashboard Reference
+menuText: Dashboard Reference
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/dashboard/)
+
+<!-- DOCS-SITE-LINK:END -->
+
+# Serverless Dashboard
+
+The [Serverless Framework Dashboard](https://dashboard.serverless.com/) is a SaaS solution that augments the Serverless Framework open source CLI to provide a powerful, unified experience to develop, deploy, test, secure and monitor your serverless applications. 
+
+The Serverless Framework free tier gives developers full access to all features included in the Serverless Framework Dashboard, but is limited to 1,000,000 function invocations per month. If you are interested in expanding your usage of the Serverless Framework beyond the free tier [contact us](https://serverless.com/enterprise/contact/) for details re available plans and pricing.
+
+## Minimum Version Requirements
+
+To take full advantage of the Serverless Framework Dashboard you must be using Serverless Framework open-source CLI version 1.48.0 or later.
+
+## Supported Runtimes and Regions
+
+Currently, the Serverless Framework Dashboard supports the following providers, runtimes and regions.  
+
+- Provider: AWS
+- Runtimes: Node.js
+- Regions: us-east-1, us-east-2, us-west-2, eu-central-1, eu-west-1, eu-west-2, ap-northeast-1, ap-southeast-1, ap-southeast-2
+
+Need unsupported providers, runtimes or regions?  Let us know what you need.  We are working hard to quickly expand our supported runtimes and regions. 
```

Comment:
"Need unsupported providers, runtimes or regions?  Let us know what you need"

Should this link out to the contact us page (or) the channel through which they can reach out to us?

Reply:
Done